tests/unit/tls_test.cc

```diff
@@ -158,20 +158,20 @@ SEASTAR_TEST_CASE(test_simple_x509_client) {
 
 SEASTAR_TEST_CASE(test_x509_client_with_builder) {
     return with_https_server_and(tls::credentials_builder{}, [](socket_address addr, std::string cert, auto& b) {
-            return b.set_x509_trust_file(cert, tls::x509_crt_format::PEM).then([&b, addr] {
-                return connect_to_ssl_addr(b.build_certificate_credentials(), addr);
-            });
+        return b.set_x509_trust_file(cert, tls::x509_crt_format::PEM).then([&b, addr] {
+            return connect_to_ssl_addr(b.build_certificate_credentials(), addr);
+        });
```

Comment:
Should be done, but seems unrelated to the overall patch set.

Reply:
it's just how the certs are built. if we stick to system CA certs, this change would not be necessary. otherwise, we need to use `tls::credentials_builder`.

Reply:
Sorry, I thought this was re-indenting old code.